### DISCUSSION TITLE
Merge with equinox.binaries?

### DISCUSSION THREAD

**Discussion by laeubi** [eclipse-equinox/equinox#409] [Ideas]
For SWT there was some initiative by @HannesWell to use LFS to have the binaries next to the code.

For equinox it might also be good to merge [equinox.binaries](https://github.com/eclipse-equinox/equinox.binaries) here using LFS, as an alternative maybe use git modules, currently this repo requires special setups (FYI @akurtakov ) as one can see in the [Jenkinsfile](https://github.com/eclipse-equinox/equinox/blob/master/Jenkinsfile) and [build.yml](https://github.com/eclipse-equinox/equinox/blob/master/.github/workflows/build.yml).

**Comment by HannesWell** [comment]
See also https://github.com/eclipse-equinox/equinox.binaries/issues/1.
But in general I think it would be possible too, if it is desired by all.

One downside in general is that extra purchased LFS bandwidth and storage budget cannot (as far as I know) be shared between organizations. In case the free budget isn't sufficient we have to purchase it for Equinox as well, just like it will probably be necessary for SWT (i.e. eclipse-platform). The same applies in case the jdt.binaries are stored in a LFS in the future.
So for each orga the 5$ a month are probably necessary (not that it would ruin the IDE working group, but I wanted to mention that).

**Reply by iloveeclipse** [reply]
> The same applies in case the jdt.binaries are stored in a LFS in the future.

Just a note: if I recall this right, jdt.binaries is only needed for **some** (disabled?) performance tests and they were not really changed for a decade. See https://github.com/eclipse-jdt/eclipse.jdt.core.binaries/commits/master

So the best way to deal with them is to remove, not to migrate.
